docs/src/administration/web/configure-environment.md

```diff
@@ -1,153 +1,133 @@
 ---
-title: "Environment configuration"
+title: Configure environments
```

Comment:
I liked the approach to having a "reference" for the most important sections of the UI, but you are right, it's probably fine to leave it out.

To avoid confusion shouldn't everything [Manage projects](https://pr-2237-xexrsbi-652soceglkw4u.eu-3.platformsh.site/overview/projects.html) be in Administration instead of the big picture?

Reply:
I moved it there because I hope we're getting rid of Administration soon. If not, yes, it would make sense.